README.rst

```diff
@@ -0,0 +1,138 @@
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-itertools/badge/?version=latest
+    :target: https://circuitpython.readthedocs.io/projects/adafruit_itertools/en/latest/
+    :alt: Documentation Status
+
+.. image:: https://img.shields.io/discord/327254708534116352.svg
+    :target: https://discord.gg/nBQh6qu
+    :alt: Discord
+
+.. image:: https://travis-ci.com/adafruit/Adafruit_CircuitPython_itertools.svg?branch=master
+    :target: https://travis-ci.com/adafruit/Adafruit_CircuitPython_itertools
+    :alt: Build Status
+
+Python's itertools for CircuitPython
+
+
+Dependencies
+=============
+This driver depends on:
+
+* `Adafruit CircuitPython <https://github.com/adafruit/circuitpython>`_
+
+Please ensure all dependencies are available on the CircuitPython filesystem.
+This is easily achieved by downloading
+`the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
+
+Installing from PyPI
```

Comment:
I would remove the entire `Installing from PyPI` section, since `itertools` is available in CPython.

Reply:
Done